cmd/libs/go2idl/go-to-protobuf/protobuf/cmd.go

```diff
@@ -68,7 +68,7 @@ func New() *Generator {
 			`+k8s.io/apimachinery/pkg/runtime`,
 			`k8s.io/apimachinery/pkg/apis/meta/v1`,
 			`k8s.io/apiserver/pkg/apis/example/v1`,
-			`k8s.io/kube-aggregator/pkg/apis/apiregistration/v1alpha1`,
+			`k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1`,
```

Comment:
not a thing to change in this PR but I think mixing api group sources in the client generator default list like this is a bad precedent.